### ISSUE TITLE
Removed from Chrome and Chrome Webstore for policy violations? 

### ISSUE THREAD

**Issue by mkgtu** [houtianze/libby-audiobook-exporter-browser-extension#10]
Was just informed by someonev today that this extension was removed from their Chrome browser for violating the Chrome Webstore policies. I verified that myself. It is no longer in the Chrome Webstore, though the "legacy" version remains available (for now). I myself don't use the extension on a desktop browser. I only use it in the Kiwi, Lemur, and Yandex mobile browsers (android), where today this extension and the legacy version both remain installed. I suppose these browsers are beyond Google's reach  Any updates/clarifications on this situation?

**Comment by houtianze** [comment]
I was informed today also.. by Google. Yes, it's removed because someone from Overdr*ve, Inc filed a claim against it (It `has an officially registered trademark to "Libby" in this context.`, etc), and Google took the following action: Action as a consequence: Approved (complete removal).

**Comment by mkgtu** [comment]
I guess they missed the legacy version, which is still in the store (for now at least). But the extension (using files from Github) could still be installed (side loaded) into chromium browsers that provide for side loading (most do, I think even Chrome itself in developer mode). 
I use only the Kiwi, Lemur, and Yandex mobile browsers with this extension and it remains installed in all three today, beyond Google's reach. Mightt help, at least for the "tech challenged" among us, if you could add a description somewhere on the GitHub page explaing which files etc should be downloaded (for the "pop-up" and/or legacy versions) in order to side load either version into browsers that allow it. There are so many who have come to rely on this. It would be a shame to lose access.

**Comment by mkgtu** [comment]
Upon further examination, I was at least partially misinformed by my original source. In fact, though the pop-up version was indeed removed from the Webstore, it WAS NOT REMOVED/DELETED from any browsers. (I'm guessing Google can't do that?) On my desktop Chrome and Edge, and in my android mobile Lemur and Yandex browsers, the extension 2.4 was simply switched off (disabled) with the warning/explanation that the extension violates Webstore policies. It was a simple matter to slide the switch back on and the extension works fine. For some reason the 2.4 extension was not switched off in the Kiwi mobile browser, nor was there any violation warning.
   Just to be safe for the future, I used Chrome's "pack extension" feature to create crx files for both currently installed versions (pop-up and legacy).
   But it would not surprise me at all if in the near future Overdrive took further steps to change the Libby website to further obfuscate files or otherwise block these extensions.
   I find Overdrive's reaction a little surprising though since they still provide step by step instructions for how to download these same audiobooks using their Overdrive apps and then transfer them to a third party more player. Go figure.

**Comment by houtianze** [comment]
> Mightt help, at least for the "tech challenged" among us, if you could add a description somewhere on the GitHub page explaing which files etc should be downloaded (for the "pop-up" and/or legacy versions) in order to side load either version into browsers that allow it. There are so many who have come to rely on this. It would be a shame to lose access.

Yes, I'm thinking about this also, maybe I will update the `READM.md` file in some time.

> they still provide step by step instructions for how to download these same audiobooks using their Overdrive apps and then transfer them to a third party more player.

I'm surprised to learn this as well, maybe their main concern is using the word `Libby` in the extension name, but who knows.

**Comment by ViniciusMRocha** [comment]
Does anyone know how to download it directly from Libby?

**Comment by mkgtu** [comment]
Download directly from Libby? Yes, obviously, in the Libby app, for use/playback in the Libby app. Other than that, NO. That would violate publishers' copyrights. And let's not kid ourselves, that's probably what this extension does. The only wiggle room room, legally, is that overdrive. does the same thing itself through its own apps, publishers be damned.
  It's just that this extension is a much more elegant way of doing the same thing.

**Comment by houtianze** [comment]
Update: I've updated the README.md with sideloading (developer mode) instruction. It's of a bit hassle, but one-time effort.
